.github/ISSUE_TEMPLATE/documentation-bug.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Documentation bug
+about: Report wrong or missing information with the documentation in the repo.
+title: "[doc] Brief description of missing or incorrect information"
+labels: 'type: docs'
+assignees: ''
+
+---
+
+<!--
+
+Thanks for stopping by to let us know something could be better!
+
+**PLEASE READ**: If you have a support contract with Google, please create an 
+issue in the [support console](https://cloud.google.com/support/) instead of 
+filing on GitHub. This will ensure a timely response.
+
+Please run down the following list and make sure you've tried the usual "quick fixes":
+
+  - Search the issues already opened: https://github.com/GoogleCloudPlatform/cloudsql-proxy/issues
+  - Check for answers on StackOverflow: http://stackoverflow.com/questions/google-cloud-sql
+
```

Comment:
- Check common proxy connectivity problems and solutions: https://cloud.google.com/sql/docs/mysql/sql-proxy#troubleshooting